hash/mz700_cass.xml

```diff
@@ -52,7 +67,7 @@ Easidata (English Version)
 
 <!-- This is a straight recording of the tape. It might be preferrable to convert to some cooked format. Is MZF considered adequate for preservation? -->
```

Comment:
This note is outdated (and repeated below a couple extra times). Mind if you remove those? It's outdated opinion, and migration of these .wav to .flac is pretty self explainatory by just grep the hash folder.

Reply:
- Updated `supported` flag regarding the PCG-700 supported or required
- Removed outdated comments